Troubleshooting.md

```diff
@@ -51,3 +51,59 @@ there are two possible reasons for this:
 2. Firebase iOS SDK also swizzles AppDelegate methods when configured on your iOS app. If after disabling notifee,
    if the delegates are still not called, this may be the reason. Method swizzling can be turned off by following
    [Firebase's documentation](https://firebase.google.com/docs/cloud-messaging/ios/client).
+
+### Deep links in push notifications not getting sent over to React Native layer when the iOS app is terminated
+
+When the iOS app is terminated (killed from the app switcher by swiping up), there is a bug in React Native that 
```

Comment:
not sure how much time itll take but perhaps itll be worth going thru a few versions of RN and call out which versions have this bug

Reply:
Yeah I did think of that but none of the react native issue threads that I've seen so far don't seem to have that info. In the absence of that we need to manually test this on older versions to find which as you eluded is going to take a while 😢  Going to merge this in as I think what we have here is indeed helpful for folks with the issue.